toolshed/routes/frontend/frontend.index.ts

```diff
@@ -3,9 +3,18 @@ import * as path from "@std/path";
 import { createRouter } from "@/lib/create-app.ts";
 import { applyProxy } from "./frontend.proxy.ts";
 import { applyStatic } from "./frontend.static.ts";
+import { cors } from "@hono/hono/cors";
 
 const router = createRouter();
 
+router.use(
```

Comment:
This is specifically for our null-origin iframes requesting `//module/charm/sandbox/bootstrap.js` as an ES module (maybe worth commenting)